e2e/pages/angularjs-material-tabs-page.ts

```diff
@@ -0,0 +1,31 @@
+import type { Locator, Page } from '@playwright/test';
+
+export class AngularjsMaterialTabsPage {
+  readonly page: Page;
+  readonly tabTwo: Locator;
+  readonly tabTwoContent: Locator;
+  readonly tabThree: Locator;
+  readonly tabThreeContent: Locator;
+  readonly buttons: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.tabTwo = page.getByRole('tab', { name: 'Tab two' });
+    this.tabTwoContent = page.getByTestId('tab-two-content');
+    this.tabThree = page.getByRole('tab', { name: 'Tab three' });
+    this.tabThreeContent = page.getByTestId('ngjs-tab-three-content');
+    this.buttons = page.getByRole('tab', { name: 'Buttons' });
+  }
+
+  async clickOnButtonsTab(): Promise<void> {
+    await this.buttons.click();
+  }
+
+  async clickOnTabTwo(): Promise<void> {
+    await this.tabTwo.click();
+  }
+
+  async clickOnTabThree(): Promise<void> {
+    await this.tabThree.click();
+  }
```

Comment:
Here too, I don't think these methods add much value, but I don't feel strongly about it.